apps/staking/src/components/AccountSummary/index.tsx

```diff
@@ -74,7 +74,7 @@ export const AccountSummary = ({
     <Image
       src={background}
       alt=""
-      className="absolute -right-40 hidden h-full object-cover object-right [mask-image:linear-gradient(to_right,_transparent,_black_50%)] md:block"
+      className="absolute -right-40 hidden h-full object-cover object-right [mask-image:linear-gradient(to_right,_transparent,_black_50%)] sm:block"
```

Comment:
Unrelated change but just something I noticed looked odd, the account summary background wasn't showing up on small screens where it should have been